### PR TITLE
Add okteto_namespace envvar to okteto stack deploy

### PR DESCRIPTION
**Pull request by jLopezbarb**
Signed-off-by: Javier López Barba <javier@okteto.com>

## Proposed changes

- Adds the option to deploy the stack from env var OKTETO_NAMESPACE
-

### CODE REVIEW

File: pkg/model/stack.go [hunk 1]
```diff
@@ -369,6 +370,10 @@ func validateStackName(name string) error {
 //UpdateNamespace updates the dev namespace
 func (s *Stack) UpdateNamespace(namespace string) error {
 	if namespace == "" {
+		envNamespace := os.Getenv("OKTETO_NAMESPACE")
+		if s.Namespace != "" && s.Namespace != envNamespace {
+			return fmt.Errorf("the namespace in the okteto stack manifest '%s' does not match the environment variable OKTETO_NAMESPACE '%s'", s.Namespace, namespace)
+		}
```
Comment (pchico83) [inline pkg/model/stack.go:376]: should we do something like `os.SetEnv("OKTETO_NAMESPACE", s.Namespace)` at some point?

Comment (jLopezbarb) [reply pkg/model/stack.go:376]: I do't think so, we don't get `OKTETO_NAMESPACE` from other place

File: pkg/model/stack.go [hunk 2]
```diff
@@ -246,6 +247,14 @@ func getStackName(name, stackPath, actualStackName string) (string, error) {
 
 // ReadStack reads an okteto stack
 func ReadStack(bytes []byte, isCompose bool) (*Stack, error) {
+	sNamespace := &StackNamespace{}
```
Comment (pchico83) [inline pkg/model/stack.go:250]: Could we do the same for okteto manifests?

**Comment by pchico83** [comment]
@jLopezbarb this has been already implemented as part of `okteto context`

**Pull request closed without merging**